src/app/game/buildings/buildings.service.ts

```diff
@@ -1,24 +1,59 @@
 import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
+import { HttpClient, HttpParams} from '@angular/common/http';
 import { Observable } from 'rxjs';
-import { environment } from '../../../environments/environment';
+import { ENDPOINTS } from '../../../environments/endpoints';
+import { Building } from '../../_models/building.model';
 
 @Injectable({
   providedIn: 'root',
 })
 
 export class BuildingsService {
 
-  constructor(private http: HttpClient) { }
+  building;
 
-  getBuildings(): Observable<any> {
-    return this.http.get('/game/buildings', { observe: 'response' });
+  constructor(private http: HttpClient) {
+  }
+
+  getBuildingsFromBackend(): Observable<any> {
+    return this.http
+      .get(ENDPOINTS.getBuildings, { observe: 'response' });
   }
 
   createBuilding(buildingType: string): Observable<any> {
-    console.log('Call to create new building made');
-    return this.http.post<any>(environment.createBuilding, {
-      type: buildingType,
-    });
+    return this.http
+      .post<any>(ENDPOINTS.createBuilding,
+                 { type: buildingType });
+  }
+
+  showFinishedBuildings() {
+    return this.http
+      .get(ENDPOINTS.getBuildings,
+           { params: new HttpParams().set('status', 'finished'), observe: 'response' });
```

Comment:
@HonzaDudek the backend won't have a parameter like that, filter the buildings in the service